helm/oauth2-proxy/README.md

```diff
@@ -60,6 +60,16 @@ See the [v1.22 API deprecations guide](https://kubernetes.io/docs/reference/usin
 
 For the same reason `service.port` was renamed to `service.portNumber`.
 
+### To 5.0.0
+
+Version 5.0.0 introduces support for custom labels and refactor [Kubernetes recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/). This is a breaking change because many labels of all resources need to be updated to stay consistent.
+
+In order to upgrade, delete the Deployment before upgrading:
+
+```bash
+kubectl delete deployment --cascade=false my-release-oauth2-proxy
```

Comment:
cascade will not work, unfortunately...

Reply:
I find this : https://github.com/codecentric/helm-charts/blob/master/charts/keycloak/README.md#from-chart-versions--500. Could you perform another test with this script @desaintmartin ?

Reply:
This should work out of the box but becomes quite complex and error-prone compared to simply delete and have a few downtime.
@pierluigilenoci what do you think?

Reply:
Maybe I don't fully understand the problem, but it could be left as a backup option only for those who don't want to have downtime. I like to make everything automatic via script but this seems a bit risky to me.

Reply:
What I propose:
- document the `kubectl delete deployment` without cascade
- as a fallback for user who don't want downtime: document this